data/low_income_thresholds.json

```diff
@@ -169,6 +169,34 @@
       }
     }
   },
+  "NV": {
+    "default": {
+      "source_url": "https://aspe.hhs.gov/topics/poverty-economic-mobility/poverty-guidelines",
+      "thresholds": {
+        "1": 30120,
+        "2": 40880,
+        "3": 51640,
+        "4": 62400,
+        "5": 73160,
+        "6": 83920,
+        "7": 94680,
+        "8": 105440
+      }
+    },
```

Comment:
this isn't really the default for NV, but it applies to the 4 nonprofits (CSA Reno, Rural Nevada Development Corporation, Nevada Rural Housing, HELP of Southern Nevada). rather than write out the same thresholds 4 times I set it as the default. if that's not the pattern we want to follow I can be more explicit

Reply:
Seems fine to me.